src/modules/iotjs_module_gpio.h

```diff
@@ -63,7 +71,12 @@ typedef struct {
   uint32_t pin;
   GpioDirection direction;
   GpioMode mode;
-#if defined(__TIZENRT__)
+  GpioEdge edge;
+#if defined(__linux__)
+  int value_fd;
+  uv_thread_t thread;
+  uv_mutex_t mutex;
```

Comment:
How big are `uv_mutex_t` and `uv_thread_t`? I think it would be better to store only pointers for them.